lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -744,17 +744,26 @@ def install_puppet_agent_from_dmg_on(hosts, opts)
         #                             Empty string if none found.
         # @api private
         def get_latest_puppet_agent_build_from_url(url)
-          require 'nokogiri'
-          require 'open-uri'
-          page = Nokogiri::HTML(open("#{url}/?C=M;O=A"))
-          agents = page.css('a').children.select{ |link| link.to_s.include? 'puppet-agent' }
-          re =  /puppet-agent-(.*)-1/
-          latest_match = agents[-1].to_s.match re
+          require 'oga'
+          require 'net-http'
```

Comment:
This fails to load for me. Changing this to the stdlib 'net/http' as documented in [stdlib](http://ruby-doc.org/stdlib-2.4.1/libdoc/net/http/rdoc/Net/HTTP.html) allows this to successfully load. Please change this to the following.

```
require 'net/http'
```